e2e/pages/events/addApplicationDocumentsEvent.page.js

```diff
@@ -34,6 +35,7 @@ module.exports = {
 
   uploadFile(file, index) {
     I.attachFile(this.fields(index).document, file);
+    I.waitForInvisible(this.fields(index).documentUploading, 20);
```

Comment:
would it be sensible to add this as part of a listener to the attach file step that then waits for the uploading text to be invisible

Reply:
good idea, I hadn't realised there were so many places `attachFile()` got called! I've moved the wait to `hooks_helper.js` rather than an event listener since it's designed for using the helper to run the steps. Thankfully it looks like all the doc upload objects in ExUI use the same "Loading..." text.

Reply:
will push up once tests pass locally

Reply:
done